Components/Avatar/avatar.css

```diff
@@ -0,0 +1,73 @@
+.avatar-sizes-list li
+{
+    font-size: 1rem;
+}
+
+.img-avatar-container
+{
+    width: 100%;
+    height: 150px;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    background-color: #F5F7F9;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 1rem;
+    box-sizing: border-box;
+}
+
+.avatar
+{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: var(--background-color);
+    background-color: var(--primary-dark-blue-color);
+    border-radius: 50%;
+}
+
+.avatar-x-large
+{
+    font-size: 4em;
+    width: 6rem;
+    height: 6rem;
+}
+
+.avatar-large
+{
+    font-size: 3em;
+    width: 5rem;
+    height: 5rem;
+}
+
+.avatar-regular
+{
+    font-size: 2em;
+    width: 4rem;
+    height: 4rem;
+}
+
+.avatar-small
+{
+    font-size: 1.5em;
+    width: 3rem;
+    height: 3rem;
+}
+
+.avatar-x-small
+{
+    font-size: 1em;
+    width: 2rem;
+    height: 2rem;
+}
+
+.avatar-img
+{
+    width: 100%;
+    height: 100%;
+    max-width: 100%;
+    max-height: 100%;
+    box-sizing: border-box;
+    border-radius: 50%;
```

Comment:
also use object-fit: cover or contain property. This would ensure that your image looks good in avatar.

Reply:
Done, thanks, now my avatar image fit properly!